includes/class-coblocks-plugin-deactivation.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * Register plugin deactivation class.
+ *
+ * @package CoBlocks
+ */
+
+// Exit if accessed directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * CoBlocks_Plugin_Deactivation Class
+ *
+ * @since NEXT
+ */
+class CoBlocks_Plugin_Deactivation {
+
+	const CONTAINER_CLASS = 'coblocks-plugin-deactivate-modal';
+
+	/**
+	 * Constructor
+	 */
+	public function __construct() {
+
+		add_action( 'admin_footer-plugins.php', array( $this, 'admin_coblocks_deactivation_modal' ) );
+
+		add_filter( 'admin_enqueue_scripts', array( $this, 'enqueue_scripts' ) );
```

Comment:
@sbalakrishnan1-godaddy Don't forget to add an empty line after this before the `}`